assignment.js

```diff
@@ -65,6 +70,11 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 2:')
 
 // Add your code below this line
+let numbersArray = [3, 48, 29, 12, 18]
+
+let currentLargestNumber = Math.max.apply(Math, numbersArray)
```

Comment:
@jimernest90 nice use of `.apply()`, it's an advanced approach, just make sure you understand it. An alternative approach would be something like this:

```
let numbersArray = [3, 48, 29, 12, 18]
let reverseSortedNumbers = numbersArray.sort().reverse()
let largestNumber = reverseSortedNumbers[0] // <- select the 1st element in the array
```